app.py

```diff
@@ -4,9 +4,38 @@
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
+
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session.get_items()
+    return render_template('index.html', items=items)
+
+@app.route('/new-item', methods=['POST'])
+def addItem():
+    title = request.form.get('title') 
```

Comment:
You could add some server side validation here

Reply:
Done 😄